test/_snapshots/formatTest_unittests.fff7ed63.0.snapshot

```diff
@@ -174,37 +175,37 @@ let addOne = x => x + 1;
 
 let result =
   addOne(0) + 0 > 1
-    ? print_string("this wont print")
-    : print_string("this will");
+    ? print_string(\"this wont print\")
+    : print_string(\"this will\");
 /*
  * Should be parsed as:
  */
 let result =
   addOne(0) + 0 > 1
-    ? print_string("this wont print")
-    : print_string("this will");
+    ? print_string(\"this wont print\")
+    : print_string(\"this will\");
 
 /*
  * Try shouldn't be aliased as ternary!
  */
 let res =
   try(something) {
-  | true => "hi"
-  | false => "bye"
+  | true => \"hi\"
+  | false => \"bye\"
   };
 
 /*
  * Many levels of if elseif should be formatted very nicely.
  */
 let result =
   if (something) {
-    Console.log("First Branch");
+    Console.log(\"First Branch\");
```

Comment:
It's weird that quotes are escaped, but I guess that's just how snapshots work.

Reply:
@bandersongit Is this intentional? It makes it harder to read snapshots to see what has actually changed.

Reply:
when no escape rely complains it cannot parse the file, same for emojis

Reply:
It looks like snapshot escaping is part of the original snapshot work @kyldvs did for the old internal TestRunner and has survived intact. I agree that not using String.escaped/emoji handling would be good to have. Curious to hear what @kyldvs's thoughts are. I can file something if we agree this is something that is worth addressing in a future release  (probably via a configuration option to not break everyone's snapshots?)

Reply:
I'm pretty sure you need to escape it for non-printable characters to have a good diff output.

There are also some crazy edge cases we ran into with snapshots and escaping with Jest, but I think they had JSON output so it might not apply here. I would just be careful when messing around with things here.